src/components/VideoComponent/VideoComponent.jsx

```diff
@@ -3,23 +3,28 @@ import './videoComponent.css';
 import { FontAwesomeIcon } from '@fortawesome/react-fontawesome'
 import { faThumbsUp, faThumbsDown, faClock, faListSquares, faCheck } from '@fortawesome/free-solid-svg-icons';
 import { useData } from '../../util-context';
-import { presentInWatchLater } from "../../util-functions/presentInWatchLater";
+import { presentInWatchLater, isVideoPresentInLiked } from "../../util-functions/";
 
 
 const VideoComponent = (video) => {
     const { state_data, dispatch_data } = useData();
-
-
-    console.log(`videoComponent`, state_data.watchLater);
     const { _id, title, source, views, dislike, liked, description, likes } = video;
 
 
     function addToWatchLaterHandler() {
-        console.log("watchWorking");
         dispatch_data({ type: "ADD_TO_WATCHLATER", payload: { video: video } })
 
     }
 
+    function addToLikeHandler() {
+        dispatch_data({ type: "LIKE_VIDEO", payload: { video: video } })
+    }
+
+    const isLiked = isVideoPresentInLiked(state_data.liked, _id);
+
+
+
+
```

Comment:
Remove this extra spacing- to make code looks good or use prettier in every file to format document.

Reply:
Removed